src/vex.js

```diff
@@ -319,7 +371,10 @@ vex.defaultOptions = {
   overlayClassName: '',
   contentClassName: '',
   closeClassName: '',
-  closeAllOnPopState: true
+  closeAllOnPopState: true,
+  giveBackFocus: true,
+  trapTabKey: true,
+  ariaContentMain: 'body:not([vex])'
```

Comment:
In the original PR, this selector was `'body > :not(.vex)'`. The selector you have selects a very different thing. What's the reason for the change?

Reply:
ariaContentMain was changed back to the correct selector